app/hotels/src/singleHotel/Stay22SingleHotel.js

```diff
@@ -0,0 +1,72 @@
+// @flow strict
+
+import * as React from 'react';
+import { graphql, PublicApiRenderer } from '@kiwicom/mobile-relay';
+
+import HotelDetailScreen from './HotelDetailScreen';
+import { type RoomConfigurationType } from '../HotelsContext';
+import { sanitizeDate } from '../GraphQLSanitizers';
+import type { Stay22SingleHotelQueryResponse } from './__generated__/Stay22SingleHotelQuery.graphql';
+
+type Props = {|
+  +goBack: () => void,
+  +currency: string,
+  +hotelId: string,
+  +checkin: Date,
+  +checkout: Date,
+  +roomsConfiguration: RoomConfigurationType,
+|};
+
+export default class Stay22SingleHotel extends React.Component<Props> {
+  getNumberOfGuests = () => {
+    return this.props.roomsConfiguration.reduce((acc, curr) => {
+      return acc + curr.adultsCount + curr.children.length;
```

Comment:
It looks like we have this function popping up in a few places. I know it is simple, but maybe extracting it would be better if ever the object structure changes.

Reply:
Yes, absolutely, I did not remember already doing this function. I will extract it to the context.